lib/job.js

```diff
@@ -33,7 +33,7 @@ function runJob(instance, job_id, request_doc, token, callback) {
     var options = ocapi.getOptions(instance, endpoint, token);
 
     // the post body
-    options['body'] = ( request_doc ? request_doc : null );
+    options['body'] = (request_doc ? request_doc : {});
```

Comment:
Fixed this, as `request` asks a **JSON-serialisable** value as body while doing json requests